doc/nrf/known_issues.rst

```diff
@@ -1849,7 +1898,7 @@ tx_buffer_length set incorrectly
 Trusted Firmware-M (TF-M)
 *************************
 
-.. rst-class:: v1-8-0
+.. rst-class:: v1-9-0 v1-8-0
 
```

Comment:
KRKNWK-12777: After soft reset TF-M will sometimes trigger a
FLASHACCERR event and halt execution.
Affects: 1.9

NCSDK-12342: When accessing protected storage a SecureFault will
sometimes trigger and execution will halt.
Affects: 1.8 and 1.9

Reply:
NCSDK-12483: Some debug symbols are sometimes missing. The current workaround is to put this in the app CMakeLists.txt: zephyr_link_libraries(-Wl,--undefined=my_missing_debug_symbol)
Affects: 1.8 and 1.9

Reply:
Hi @mia-ko , review would be appreciated.

Reply:
@SebastianBoe I have added these. Please check if any modifications are required.